client/src/components/History/CurrentHistory/HistoryCounter.vue

```diff
@@ -12,19 +12,25 @@
                 <span>{{ historySize | niceFileSize }}</span>
             </b-button>
             <b-button-group>
-                <b-button
-                    :id="`history-storage-${history.id}`"
-                    variant="link"
-                    size="sm"
-                    class="rounded-0 text-decoration-none"
-                    @click="showPreferredObjectStoreModal = true">
-                    <icon icon="hdd" />
-                </b-button>
-                <PreferredStorePopover
-                    :history-id="history.id"
-                    :history-preferred-object-store-id="historyPreferredObjectStoreId"
-                    :user="user">
-                </PreferredStorePopover>
+                <ConfigProvider v-slot="{ config }">
```

Comment:
Can you use `useConfig(true)` here please ? The ConfigProvider triggers a new fetch every time.